src/AzureAuth/Config.cs

```diff
@@ -9,7 +9,7 @@ namespace Microsoft.Authentication.AzureAuth
     using Tomlyn;
 
     /// <summary>
-    /// The config.
+    /// A config contains a collection of <see cref="Alias"/>.

```

Comment:
This is true. An array or a list might also be considered a collection though. It might be a _tiny_ bit more helpful to say that a `Config` is a mapping of names to `Alias`es.